tests/lowest_rate_test.go

```diff
@@ -0,0 +1,75 @@
+package easypost_test
```

Comment:
All of the tests in this file should instead go in the `shipment_test.go` file since they are shipment tests. We try to group all tests into a file named after the object being tested. Each object may then have a dozen or more individual tests to test different functionality related to that object.

I'm also not seeing a cassette for these tests. Is VCR working properly to record these so that we can have consistent results across test runs? AH - after looking I see you're mocking the `Rates` object, pretty clever! With that being said, we should be making a real HTTP call to create a shipment and then getting the lowest rate from that instead so that we can test this completely.

Reply:
As per our conversation, we are going to mock the `Rates` object for now.